projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1184,22 +1141,22 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     }
 
     /**
-     * @hidden
+     * Triggers change detection on the combo view
      */
     public triggerCheck() {
```

Comment:
@Lipata, @damyanpetev 
I've exposed this in the API docs, since it's used in the combo README.md, in the example for [remote data loading](https://github.com/IgniteUI/igniteui-angular/blame/master/projects/igniteui-angular/src/lib/combo/README.md#L44).